deploy/kustomize/patches/clusterserviceversion-patch.yaml

```diff
@@ -1,58 +1,50 @@
-#
-# Operator display name
-#
-- op: replace
-  path: /spec/displayName
-  value: Operator for PMEM-CSI driver
-#
-# Operator provider
-#
-- op: replace
-  path: /spec/provider
-  value:
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
```

Comment:
If this handwriting of CSV is ok then we can make it more simple I guess by using this as base manifest for operator-sdk. That way we could avoid explicit patching. I take this and try that.

Reply:
This is just a different way of specifying our custom fields, so yes, this is okay and in fact unavoidable.